textdescriptives/components/coherence.py

```diff
@@ -0,0 +1,80 @@
+from typing import List
+
+import numpy as np
+from spacy.language import Language
+from spacy.tokens import Doc
+
+
+@Language.factory("textdescriptives.coherence")
+def create_coherence_component(nlp: Language, name: str):
+    """Allows Coherence to be added to a spaCy pipe using nlp.add_pipe("textdescriptives.coherence").
+    If the pipe does not contain a parser or sentencizer, the sentencizer component is silently added."""
```

Comment:
```suggestion
    If the pipe does not contain a parser or sentencizer, the sentencizer component is silently added
    """
```

Isn't this the user-facing API. I would add args, return and as well as an example.

Reply:
Artefact from copy-pasting from a different class - good catch :D 

Honestly unsure what the user facing api is. The user would never call neither `create_coherence_component` nor instantiate the `Coherence` class. 
Does spacy write their documentation in the `create_*` functions? if so, I agree - let's move it there @KennethEnevoldsen